docs/pages/reference/api/getting-started.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: API Getting Started Guide
+description: Getting started with the Teleport API
+---
+
+# Getting Started
+
+In this getting started guide we will use the Teleport API Go Client to connect to the Teleport Auth server.
+
+Here are the steps we'll walk through:
+ 1. Create an API user
+ 2. Generate credentials
+ 3. Create a Go program to demo the client
+
+<Admonition type="warning">
+  This quickstart connects to the Teleport web proxy with an insecure connection. In production, 
```

Comment:
Let's always use secure connection in all guides. You can leave it to the user to use insecure mode if they need it. The danger that is that people will copy-paste the guide and will forget to remove the insecure flags.